lab0/README-zh.md

```diff
@@ -0,0 +1,65 @@
+# Map-Reduce Task
+
+[English Ver](./README.md)
+
+## 介绍
+
+我们为 VLDB Summer School 2021 设置了一个 Map-Reduce 的小作业。通过完成本课程，你可以学习到一些 Golang 的基础知识并对分布式系统有一个基础的了解。本课程与 MIT 6.824 的 [lab1](https://pdos.csail.mit.edu/6.824/labs/lab-mr.html) 相似，适合作为学习分布式系统的第一步。
+
+本课程提供了一个未完成的 Map-Reduce 框架，你需要完成这个框架，并且使用它从文件中找到出现频率最高的 10 个 URL。
+
+## 课程
+
+我们推荐 MIT 6.824 的第一课（介绍章节）来学习课程。MIT 6.824 对分布式系统进行了整体介绍并且详细的讲解了 Map-Reduce 框架。
+
+- [Official] https://www.youtube.com/watch?v=cQP8WApzIQQ
+- [Mirror] https://www.bilibili.com/video/BV1R7411t71W?p=1
+
+Map-Reduce 是一个著名的分布式计算框架，除了 MIT 6.824，可以参考网上找到的其他资料来帮祝你学习，
+
+对于 Golang 的初学者，我们推荐先通过 [Online Go tutorial](https://tour.golang.org/) 学习语言。
+
+在 MIT 6.824 的课程指南中，还有很多学习建议，例如如何对复杂系统进行 Debug，推荐感兴趣的同学阅读。
+
+- https://pdos.csail.mit.edu/6.824/labs/guidance.html
+
+## 向导
+
+这项课程的目标是能够使用 Golang 开发分布式系统。此外，你需要使用 GitHub 来完成课程的流程。
+
+- Fork 本仓库到你的帐号下
+- 在自己的仓库中完成课程
+- 向本仓库发送 Pull Request
```

Comment:
We may receive more than 100 pull requests here by this. Does we really need it?